qemu/tests/cfg/block_during_io.cfg

```diff
@@ -48,7 +48,10 @@
     variants:
         - iozone_stress:
             stress_name = iozone
-            iozone_cmd_opitons = "-azR -r 64k -n 1G -g 10G -M -i 0 -i 1 -I "
+            Win10:
+                iozone_cmd_opitons = "-azR -r 64k -n 1G -g 5G -M -i 0 -i 1 -I "
+            Win8, Win8.1, Win2012, Win2012..r2, Win2016, Win2019:
+                iozone_cmd_opitons = "-azR -r 64k -n 1G -g 10G -M -i 0 -i 1 -I "
```

Comment:
```suggestion
            iozone_cmd_opitons = "-azR -r 64k -n 1G -g 10G -M -i 0 -i 1 -I "
            Win10:
                iozone_cmd_opitons = "-azR -r 64k -n 1G -g 5G -M -i 0 -i 1 -I "
```

Reply:
Thanks, have update it

(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.block_during_io.iozone_stress.system_disk.send_qmp.with_system_reset.q35: PASS (829.44 s)

related log:
21:46:10 DEBUG|     iozone_cmd_opitons = -azR -r 64k -n 1G -g 5G -M -i 0 -i 1 -I -f %s:\testfile